pages/bindingDescription.xml

```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <p>You can always further specify this information within one
         of the <gi>decoNote</gi> elements within
         <gi>binding</gi>.</p>
-        </body>
+
+    <div type="level3">
```

Comment:
I would add an introductory line like 'Here are some examples of how to use typed decoNote elements to provide specific information'

Reply:
see my commit